### DISCUSSION TITLE
Welcome to IBM-QRadar-Universal-Cloud-REST-API Discussions!

### DISCUSSION THREAD

**Discussion by ChrisCollinsIBM** [IBM/IBM-QRadar-Universal-Cloud-REST-API#135] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by JiaChee** [comment]
Hello,

Does anyone has idea that whether Universal Cloud RESTAPI is able to support Chinese characters via POST Method. For GET method, it is no issue to capture Chinese characters. @ChrisCollinsIBM

**Comment by ghost** [comment]
Hi all,


We have a DMARC solution called DMARCLY ([DMARCLY | Email Security, Authentication, Anti-Phishing, SPF, DKIM, DMARC Monitoring, and DMARC Analyzer](https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdmarcly.com%2F&data=05%7C02%7C%7Ca4f6ba8f7bd34cf815e608dd64c53165%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C638777520295464982%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=ENGAapaLQWPO9KGtDqX%2FOMMZX2HnsMzMfHklQYYFJoU%3D&reserved=0)) we're trying to pull logs from into QRadar SIEM using the Universal Cloud REST API protocol.

When I run the test (from QRadar CLI) on my workflow and workflow parameters, I get the error shown below in the capture. 

![test_error_message](https://github.com/user-attachments/assets/42d3a0c6-c88e-4e33-8e59-d57bd9bf37c3)


However, when I test by directly querying the API endpoint using the "curl" command, it works as shown in the screenshot below.

![test_success_with_curl_command](https://github.com/user-attachments/assets/d363a56f-8fe9-4fcf-9a0c-ebc4575d7607)



I've been struggling with that for months. I'm stuck and completely clueless. I'm a total newbie in this API workflow matter, having never touched it before.

Can anybody please help me figure this out? 

I pasted below the workflow and the workflow parameters I'm running.

Thanks,




---workflow---

<?xml version="1.0" encoding="UTF-8" ?>
<Workflow name="DMARCLY" version="1.0" xmlns="http://qradar.ibm.com/UniversalCloudRESTAPI/Workflow/V2">
 <Parameters>
    <Parameter name="api_key"        label="DMARCLY secret key"           required="true" secret = "true"/>
    <Parameter name="dmarcly_host"       label="https://dmarcly.com/api"    required="true" />
  </Parameters>
    <Actions>

        <CallEndpoint url="https://dmarcly.com/api/aggregate_by_unaligned" method="GET">

                    <SSLConfiguration allowUntrustedServerCertificate="true" />
                        <BearerAuthentication token="${/access_token}" />

        </CallEndpoint>

    </Actions>

</Workflow>


---parameters---

<?xml version="1.0" encoding="UTF-8" ?>
<WorkflowParameterValues xmlns="http://qradar.ibm.com/UniversalCloudRESTAPI/WorkflowParameterValues/V2">
        <Value name="api_key"        value="fc600bb3c894d5497b61ea2145ae79f0"/>
                <Value name="dmarcly_host"       value="https://dmarcly.com/api"/>
</WorkflowParameterValues>

 

 

 

 

 

 

Thanks,

**Reply by ghost** [reply]
my workflow & parameters:

---workflow---

<?xml version="1.0" encoding="UTF-8" ?>
<Workflow name="DMARCLY" version="1.0" xmlns="http://qradar.ibm.com/UniversalCloudRESTAPI/Workflow/V2">
 <Parameters>
    <Parameter name="api_key"        label="DMARCLY secret key"           required="true" secret = "true"/>
    <Parameter name="dmarcly_host"       label="https://dmarcly.com/api"    required="true" />
  </Parameters>
    <Actions>

        <CallEndpoint url="https://dmarcly.com/api/aggregate_by_unaligned" method="GET">

                    <SSLConfiguration allowUntrustedServerCertificate="true" />
                        <BearerAuthentication token="${/access_token}" />

        </CallEndpoint>

    </Actions>

</Workflow>

---parameters---

<?xml version="1.0" encoding="UTF-8" ?>
<WorkflowParameterValues xmlns="http://qradar.ibm.com/UniversalCloudRESTAPI/WorkflowParameterValues/V2">
        <Value name="api_key"        value="fc600bb3c894d5497b61ea2145ae79f0"/>
                <Value name="dmarcly_host"       value="https://dmarcly.com/api"/>
</WorkflowParameterValues>
